src/quests/keys.ts

```diff
@@ -0,0 +1,60 @@
+import { availableAmount, cliExecute, restoreMp, useSkill } from "kolmafia";
+import {
+  $item,
+  $items,
+  $location,
+  $skill,
+  adventureMacro,
+  get,
+  have,
+  maximizeCached,
+  sum,
+} from "libram";
+
+import { Macro } from "../combat";
+import { propertyManager } from "../global";
+import { acquire } from "../lib";
+import { moodBaseline } from "../mood";
+
+function keyCount() {
+  return sum($items`fat loot token, Boris's key, Jarlsberg's key, Sneaky Pete's key`, (item) =>
```

Comment:
If we somehow have three boris's keys, this returns 3. Is that what we want?

Reply:
fair enough